src/help.js

```diff
@@ -0,0 +1,25 @@
+import shell from 'shelljs';
+
+const commandList = Object.keys(shell)
+  .filter(cmd => typeof shell[cmd] === 'function' && cmd !== 'ShellString');
+
+const help = [
```

Comment:
Can use a template string instead, they are multi-line:

```js
const help = `
  shx: A wrapper for shelljs UNIX commands.
  
  Usage: shx <command> [options]

  Example:

      $ shx ls .
      foo.txt
      bar.txt
      baz.js

      $ shx rm -rf *.txt
      $ shx ls .
      baz.js

  Commands:

${(commandList.map(cmd => '    - ' + cmd)).join('\n')}
`;
```

I added a Usage: part in there, feel free to rework this.